src/frontend/tsconfig.json

```diff
@@ -17,12 +17,15 @@
     ],
     "module": "esnext",
     "moduleResolution": "node",
+    "noImplicitAny": true,
+    "noImplicitThis": true,
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
+    "strictNullChecks": true,
     "sourceMap": true,
     "target": "es6",
-    "types": ["jest"]
+    "types": ["node"]
```

Comment:
Why ?

Reply:
I use require in the tests and if I don't change jest config, it returns me this error, 7 times as I use require 7 times
Cannot find name 'require'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node` and then add `node` to the types field in your tsconfig.
require('.').default